tests/test_schema_exporter.py

```diff
@@ -215,6 +250,3 @@ def test_export_metamodel_slots():
     # of this will change
     examples = s['examples']
     assert 'bibo:draft' == examples
-
-
-
```

Comment:
we need to start with tox for this repo!

Reply:
Ok, I watched a video about tox last night, but I don't think I'm ready to make that change yet.